src/components/Form/Form.jsx

```diff
@@ -183,6 +196,22 @@ const Form = () => {
                 <MenuItem value="Sulaymaniyah">Sulaymaniyah</MenuItem>
                 <MenuItem value="Wasit">Wasit</MenuItem>
               </Select>
+              <h4 className={classes.title}>Neighborhoods/Places</h4>
+              <p className={classes.desc}>
+              Please provide an approximate location 
+                (Bar, Restaurant, Park, etc.)/(Neighborhoods)
+              </p>
+              <TextField
+                id="outlined-basic"
+                label="Neighborhoods/Places"
+                variant="outlined"
+                size="large"
+                className={classes.input}
+                value={neighbor}
+                onChange={(e) => setNeighbor(e.target.value)}
+                required
+              />
```

Comment:
Since this is a form, why there isn't any validation of data here? 
I would suggest using `yup` or `Joi` for this!